aspnetcore/performance/caching/distributed.md

```diff
@@ -226,7 +226,7 @@ The <xref:Microsoft.Extensions.Caching.Distributed.IDistributedCache> interface
 
 ## Establish distributed caching services
 
-Register an implementation of <xref:Microsoft.Extensions.Caching.Distributed.IDistributedCache> in `Startup.ConfigureServices`. Framework-provided implementations described in this topic include:
+Register an implementation of <xref:Microsoft.Extensions.Caching.Distributed.IDistributedCache> in `Program.cs`. Framework-provided implementations described in this topic include:
```

Comment:
Revert all the changes  from here down.  You only make changes to the the .NET 6 version between
```
::: moniker range=">= aspnetcore-6.0"
// make changes here
::: moniker-end
```

Everything below `::: moniker range="< aspnetcore-6.0"` is for the versions lower than 6.

Reply:
done